README.md

```diff
@@ -99,77 +103,110 @@ docker run \
+6. Invoke the load operation completion handler.
 
 ### 4.1.3. Update
 
-Not supported. Stream append may be performed using `create` load operation type.
+Not supported. A stream append may be performed using the `create` load operation type and a same stream previously used
+to write the events.
 
 ### 4.1.4. Delete
 
 Not supported.
 
-## 4.2. Path Item Operations
+## 4.2. Stream Operations
 
 Mongoose should perform the load operations on the *streams* when the configuration option `item-type` is set to `path`.
 
 ### 4.2.1. Create
 
-1. Check the corresponding scope if it exists and create it if not.
-2. Invoke `StreamManager.createStream` using *item path* as a *scope name* and *item id* as a *stream name*. Fail the
-load operation using the status code *14* if the method returns `false`.
+**Notes**:
+> * Just creates empty streams
+> * Works synchronously
+
+Steps:
+1. Get the endpoint URI from the cache.
+2. Check if the corresponding `StreamManager` exists using the cache, create a new one if it doesn't.
+3. Check if the destination scope exists using the cache, create a new one if it doesn't.
+4. Create the corresponding stream using the `StreamManager` instance, scope name, etc.
+5. Invoke the load operation completion handler.
 
 ### 4.2.2. Read
 
-For each item read the whole corresponding stream. Unlike events, it's possible to read the streams in the random order
-so both `item-input-path` (scope's streams listing) and `item-input-file` (stream ids list) options should be supported.
+Read the whole streams (all the corresponding events)
+
+Steps:
+1. Get the endpoint URI from the cache.
+2. Check if the corresponding `StreamManager` exists using the cache, create a new one if it doesn't.
+3. Check if the corresponding `ClientFactory` exists using the cache, create a new one if it doesn't.
+4. Check if the corresponding `EventStreamReader<ByteBuffer>` exists using the cache, create a new one if it doesn't.
+5. Read all events in the stream in the loop, discard the returned byte buffers content.
+6. Invoke the load operation completion handler.
 
-There is also another option, called `reader-timeout`. Pravega documentation says it only works when there is no available Event 
-in the stream. `readNextEvent()` will block for the specified time in ms. So, in theory 0 and 1 should work just fine. 
-They do not. In practice, this value should be somewhere between 100 and 2000 ms (2000 is Pravega default value).
 ### 4.2.3. Update
 
 Not supported
 
 ### 4.2.4. Delete
 
-A deletion of a path composed of a scope and a stream is implemented through `invokePathDelete()` method. Before the deletion, the stream must be sealed because of Pravega concepts. So the sealing of the stream is done in `invokePathDelete()` method too.
+Before the deletion, the stream must be sealed because of Pravega concepts. So the sealing of the stream is done during
+the deletion too.
```

Comment:
The gap here is meaningful and should be filled with data later? or just a misprint?

Reply:
not sure. which gap?